aequilibrae/project/network/network.py

```diff
@@ -256,12 +268,17 @@ def set_time_field(self, time_field: str) -> None:
 
     def count_links(self) -> int:
         c = self.conn.cursor()
-        c.execute("""select count(*) from links""")
+        c.execute("""select count(link_id) from links""")
```

Comment:
These could all be count(*) with a where condition

```python
    def count(tablename, where_condition=""):
        c = self.conn.cursor()
        return c.execute(f"select count(*) from {tablename} {where_condition};").fetchone()[0]

Reply:
The count(*) worked fine too, but I wanted to make it a little more readable. That's all

Reply:
Oh. Now I see the point. A single method for all of them...

Reply:
#noqa for the win in these situations